utils/keycodes.js

```diff
@@ -90,3 +90,19 @@ export const displayShortcut = mapValues( modifiers, ( modifier ) => {
 		return shortcut.replace( /⌘\+([A-Z0-9])$/g, '⌘$1' );
 	};
 } );
+
+export const is = mapValues( modifiers, ( getModifiers ) => {
+	return ( event, character, _isMac = isMacOS ) => {
+		const mods = getModifiers( _isMac );
```

Comment:
Could we call this `modifiers` instead of `mods`? Explicit variable names are nice and we can afford the bytes 😉

Reply:
That would conflict with `modifiers` higher up.

Reply:
Oh, duh, sorry about that. 👍